package.json

```diff
@@ -10,7 +10,7 @@
   "scripts": {
     "build": "npm run clean && better-npm-run build",
     "build:blog-utils": "npm run clean && bin/create-package-json-for-blog-utils && better-npm-run build:blog-utils",
-    "build:blog-utils-dev": "NODE_ENV=development npm run build:blog-utils -- --watch",
+    "build:blog-utils-dev": "NODE_ENV=development concurrently 'npm run build:blog-utils -- --watch' 'bin/serve-blog-utils'",
```

Comment:
I'm probably missing something about how `concurrently` works (I assumed from the naming it runs the commands simultaneously), but how can we serve blog-utils before we're finished building it?

Reply:
The node server serves the dist files. If you browse the dev page as soon as it is up, the files will fail to load in the browser until these files are generated by webpack.